### PR TITLE
basic UI

### PR DESCRIPTION
**Pull request by Obimbo07**
In this PR the following changes were implemented:

- Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.

- In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    list_rental
+  else
+    puts 'Invalid option. Please choose an option 1 - 7 and try again.'
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
```
Comment (danmainah) [inline main.rb]: - [ ] @Obimbo07 Great work adding the UI, however, the UI doesn't get displayed in the console, kindly check your implementation of the prompts, if using a method you should call them for the UI to get displayed( in your case you can call prompt method to display the menu list).

- [ ] After implementing the changes requested kindly ensure the UI matches the design given from the sneak peek video and all menu options should work correctly.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Obimbo07 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Implemented the App class that has the functionality to add new persons and list persons add books and list books, add rentals, and list rentals.
✔️  Added entry point for the app in main.rb file with a method to list options.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  end
+end
+
+def call_option(option)
+  case option
+  when 1
+    list_books
+  when 2
+    list_people
+  when 3
```
Comment (Ghiftee) [inline main.rb:31]: - [ ] Great job working on the project thus far! However, when I try to create a `student` or `teacher`, I get this error and cannot successfully create a person or test a rental.
Screenshot 2023-12-08 at 20 47 57

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there @Obimbo07 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
✔️ The Ui is displayed in the console
✔️ A book can be created successfully
✔️ No linter errors
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Obimbo07 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Implemented requested changes ✔️
- UI works as expected ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**